src/plugins/cosmos/components/modals/Staked/views/Staked.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, ModalCloseButton } from '@chakra-ui/react'
+import { Asset } from '@shapeshiftoss/types'
+import { AnimatePresence } from 'framer-motion'
+import { RewardsRow } from 'plugins/cosmos/components/RewardsRow/RewardsRow'
+import { StakedHeader } from 'plugins/cosmos/components/StakedHeader/StakedHeader'
+import { StakedRow } from 'plugins/cosmos/components/StakedRow/StakedRow'
+import { StakingButtons } from 'plugins/cosmos/components/StakingButtons/StakingButtons'
+import { UnbondingRow } from 'plugins/cosmos/components/UnbondingRow/UnbondingRow'
+import { Text } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+type StakedProps = {
+  assetId: string
+}
+
+// TODO: Wire up the whole component with staked data
+export const Staked = ({ assetId }: StakedProps) => {
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
```

Comment:
any reason we can't add this selector now? i feel like we did this before somewhere...

Reply:
Yeah we do have `selectAssetNameById` which I originally added. The problem is we don't have anything Cosmos SDK to select from currently